apps/cookbook/src/app/showcase/list-load-on-demand-showcase/list-load-on-demand-showcase.component.html

```diff
@@ -1,4 +1,29 @@
 <div class="example">
+  <p>
+    The kirby list will emit at <code>loadMore</code> event, when the user has scrolled 80% of the
+    list. It is the responsibility of the consumer to add more items to the bound
+    <code>items</code>.
+  </p>
+  <p>
+    The event emitted is of type <code>KirbyLoadMoreEvent</code> which exposes a
+    <code>complete</code> callback function.
+  </p>
+  <p>
+    The <code>complete</code> callback function should be called when the loading of the additional
```

Comment:
Consider readability:
```suggestion
    The <code>complete()</code> callback function should be called when the loading of the additional
```